includes/Main.php

```diff
@@ -1646,8 +1646,19 @@ function_exists( 'WPO_WCPDF_Templates' ) &&
 			return;
 		}
 
-		$due_date       = apply_filters( 'wpo_wcpdf_due_date_display', date_i18n( wcpdf_date_format( $this, 'due_date' ), $due_date_timestamp ), $due_date_timestamp, $document_type, $document );
-		$due_date_title = is_callable( array( $document, 'get_due_date_title' ) ) ? $document->get_due_date_title() : __( 'Due Date:', 'woocommerce-pdf-invoices-packing-slips' );
+		$due_date = apply_filters_deprecated(
+			'wpo_wcpdf_due_date_display',
+			array(
+				date_i18n( wcpdf_date_format( $this, 'due_date' ), $due_date_timestamp ),
+				$due_date_timestamp,
+				$document_type,
+				$document
+			),
+			'3.8.7',
```

Comment:
Must be `3.9.0`

Reply:
Since this new function was introduced in version `3.8.7`, shouldn't we use this version here?

Reply:
However, the old hook is still present, and there is no indication that it was deprecated in version `3.8.7`.